### PR TITLE
Portfolio desktop version final:- module 1 

### PR DESCRIPTION
**Pull request by yesuf023**
### Desktop version portfolio milestone project:- UI
  **This project includes the following concepts 👍 :-**
  

-   Used media queries in your CSS file to make your desktop version.
-   Used Grid and Flexbox in order to lay out the elements on the page.
-   Created UI is adaptable to different screen sizes.
-   Finalize the desktop version portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,276 +6,307 @@
...
+                  <li>
+                      <textarea name="text_area" id="text_area" cols="30" rows="10" maxlength="500"
+                          placeholder="Enter text here" required></textarea>
+                  </li>
+                  <li class="submit-button">
+                      <button type="submit">
+                          Get in touch
+                      </button><br>
+                      <small></small>
+                  </li>
```
Comment (Tufahel) [inline index.html:278]: image

- [ ] This `button` should be aligned with the form as per the given design. Kindly fix this issue.

File: index.html [hunk 2]
```diff
@@ -6,276 +6,307 @@
...
+              </ul>
+          </form>
+      </div>
+      <ul class="social-media-icons">
```
Comment (Tufahel) [inline index.html:282]: image

- [ ] There should be an underline before the `social icons` at the footer to match the given design. Kindly fix this issue.

File: index.html [hunk 3]
```diff
@@ -6,276 +6,307 @@
...
+              </div>
+              <h2 class="categorie-title">Languages</h2>
+              <ul class="frame19-tags">
+                <li class="tag">HTML</li>
+                <li class="tag">JS</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Ruby</li>
+              </ul>
+            </article>
           </li>
```
Comment (Tufahel) [inline index.html:212]: image

- [ ] You have used `color: lightgray` as the background of `HTML`, `JS`,`CSS`,`Ruby & Rails`. But as per the given design guideline the `background color` should be `#EBF0EE`. Kindly fix this issue.

image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @yesuf023 :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Linter checks are passing.
✔️ Follow the correct GitHub flow.
✔️ Commit messages are meaningful.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

- [ ] ***[OPTIONAL]*** Consider using the `capital` letter of the first letter of a commit message as it is more professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ofarouq310 — APPROVED** [review]
Hi @yesuf023, ✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇  

giphy

## Highlights
✅ Followed GitHub workflow.
✅ Decriptive Pr title & summary.
✅  Meaningful commit messages.

## Optional suggestions

- You are doing a great job, and you are tired I can tell, but try to give some more time to lay out the elements on the page properly, refer to the `Figma` design specs to get a better understanding of the spacing, padding & margin you need. You have got what it takes 💪

| Your design | Figma Design |
| ------------- | ------------- |
|Screenshot 2023-06-11 014826 | image| 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**